cli/test/action/p2p_replicator_delete.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2025 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package action
+
+import (
+	"strings"
+
+	"github.com/stretchr/testify/require"
+)
+
+// P2PReplicatorDelete executes the `client p2p replicator set` command.
```

Comment:
todo: incorrect documentation, please fix.

```suggestion
// P2PReplicatorDelete executes the `client p2p replicator delete` command.
```

Reply:
done